src/assisted_test_infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -10,6 +10,9 @@
 
 @dataclass
 class BaseNodeConfig(BaseConfig, ABC):
+    # CI build id
+    build_id: str = None
```

Comment:
What is `build-id` and why do we need it for?

Reply:
to create the vsphere machines under a folder which contains this build_id

Reply:
If this is Vsphere only variable, please pass it to Vsphere on dedicated configuration or on some other way

Reply:
It isn't. It's the CI build-id. I use it now for vsphere but it can be used elsewhere

Reply:
I think it's just confusing to duplicate the values with different letter case
Any easy action that can be made to remove confusion is a good idea